src/components/Profile/components/PersonalInfoList/UpdateUserPrivacyDialog/index.jsx

```diff
@@ -0,0 +1,80 @@
+import { FormControl, IconButton, Input, InputLabel } from '@mui/material';
+import GordonSnackbar from 'components/Snackbar';
+import { useState, useEffect } from 'react';
+import userService from 'services/user';
+import SearchField from 'views/PeopleSearch/components/SearchFieldList/components/SearchField';
+
+const UpdateUserPrivacy = (username, field) => {
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  const [group, setGroup] = useState('');
+  const [groupList, setGroupList] = useState([]);
+  let visibleTo = '';
+
+  const handlePrivacy = async (event) => {
+    try {
+      await userService.setUserPrivacy({ Field: field, VisibilityGroup: event.target.value });
+      setSnackbar({
+        message: 'Your office location will update within a couple hours.',
+        severity: 'success',
+        open: true,
+      });
+    } catch {
+      setSnackbar({
+        message: 'Office location failed to update. Please contact CTS.',
+        severity: 'error',
+        open: true,
+      });
+    }
+  };
+
+  useEffect(() => {
+    userService.getVisibilityGroups().then(setGroupList);
+  }, []);
+
+  useEffect(() => {
+    userService.getPrivacySetting(username).then(setGroup);
+  }, [group]);
+
+  let tempField = field;
```

Comment:
```suggestion
  let tempField = field;
```

what is the benefit of having a temporary variable here?

Reply:
you don't ever use the `tempField` variable besides holding the `field`'s information at this very moment.